app/views/parent/kids/edit.html.erb

```diff
@@ -9,14 +9,14 @@
   <%= form.text_field :last_name %>
 
   <%= form.label :birthdate %>
-  <%= form.date_field :birthdate %>
+  <%= form.date :birthdate %>
 
   <%= form.label :sin %>
-  <%= form.text_field :sin %>
+  <%= form.number_field :sin %>
```

Comment:
Actually, leave `sin` and `medicate` as text strings; they're actually alphanumeric so would need to be stored as strings